src/tools/log-parser-rs/src/args.rs

```diff
@@ -0,0 +1,86 @@
+use clap::{Parser, ValueEnum};
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[command(name="kata-log-parser", author, version, about, long_about = None)] // Read from `Cargo.toml`
+pub struct Cli {
+    pub input_file: PathBuf,
+
+    #[arg(short, long)]
+    pub output_file: Option<PathBuf>,
+
+    #[arg(short, long, help = "check log files and only display output on error")]
+    pub check_only: bool,
+
+    //TODO NOTE:(Not sure if this is still required, or why it needs output file set. (is only here
+    //because it was an arg for the go version.)
+    #[arg(
+        short,
+        long,
+        help = "display debug information (requires '--output-file')",
+        requires = "output_file"
+    )]
+    pub debug: bool,
+
+    //TODO NOTE:currently equivilant to error-if-no-records. Will change with multifile support
+    #[arg(long, help = "error if any files are empty")]
+    pub error_if_file_empty: bool,
+
+    //TODO NOTE:the original documentation didnt mention the abillity to include multiple log
+    //files, I assume one could just include as many filnames at the end as you wanted?
+    #[arg(long, help = "error if all logfiles are empty")]
+    pub error_if_no_records: bool,
```

Comment:
so, the documentation for the go version doesnt specify that you can have multiple input files,  but looking at main.go, it seems that indeed, it can take any number of infiles. Ill add that functionality.

Reply:
Ta! ;)